automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/OrchestrationApiSpec.scala

```diff
@@ -36,7 +49,7 @@ class OrchestrationApiSpec extends FreeSpec with Matchers with ScalaFutures with
       val role = "bigquery.jobUser"
 
       val user: Credentials = UserPool.chooseStudent
-      val userToken: AuthToken = user.makeAuthToken()
+      val userToken: AuthToken = user.makeAuthToken(AuthTokenScopes.userLoginScopes ++ Seq(BigqueryScopes.CLOUD_PLATFORM))
```

Comment:
one of the changes between serviceTest 0.10 and 0.13 is that we no longer request `CLOUD_PLATFORM` scope by default when making test-user tokens (b/c we don't request it at runtime for end users). That change broke this test, so I had to update the test.

Reply:
I think you can just do `AuthTokenScopes.userLoginScopes +BigqueryScopes.CLOUD_PLATFORM`?